adoc/chapters/programming_interface.adoc

```diff
@@ -14373,6 +14373,11 @@ functions and common member functions that are listed in
 <<sec:reference-semantics>> in <<table.specialmembers.common.reference>> and
 <<table.hiddenfriends.common.reference>>, respectively.
 
+As with all SYCL objects that have the common reference semantics, kernel
+identifiers are equality comparable.  Two [code]#kernel_id# objects compare
+equal if an only if they refer to the same application kernel or to the same
```

Comment:
```suggestion
equal if and only if they refer to the same application kernel or to the same
```

Reply:
Fixed in 67abc57aed6aa818c374e044dbb387f6625c9502.